sddi-base/Dockerfile

```diff
@@ -23,228 +37,164 @@ RUN set -ex && \
-  curl -o /wheels/ckanext-spatial.txt \
-    https://raw.githubusercontent.com/MarijaKnezevic/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements.txt && \
-  pip install -r https://raw.githubusercontent.com/MarijaKnezevic/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements-postgis.txt && \
-  curl -o /wheels/ckanext-spatial-postgis.txt \
-    https://raw.githubusercontent.com/MarijaKnezevic/ckanext-spatial/${CKANEXT_SPATIAL_VERSION}/requirements-postgis.txt && \
-    ls -lah /wheels
+# ckanext-dcat ##########################################################
+ARG CKANEXT_DCAT_VERSION="v1.5.1"
+ENV CKANEXT_DCAT_VERSION=${CKANEXT_DCAT_VERSION}
+ENV CKANEXT_DCAT_GITHUB_URL="https://github.com/ckan/ckanext-dcat"
 
 RUN set -ex && \
-  pip wheel --wheel-dir=/wheels \
-    git+https://github.com/MarijaKnezevic/ckanext-spatial.git@${CKANEXT_SPATIAL_VERSION}#egg=ckanext-spatial
+    curl -o /wheels/ckanext-dcat-requirements.txt \
+        https://raw.githubusercontent.com/ckan/ckanext-dcat/${CKANEXT_DCAT_VERSION}/requirements.txt && \
+    pip install -r /wheels/ckanext-dcat-requirements.txt && \
+    pip wheel --wheel-dir=/wheels \
+        git+${CKANEXT_DCAT_GITHUB_URL}.git@${CKANEXT_DCAT_VERSION}#egg=ckanext-dcat
 
 ###############################################################################
 # Runtime stage
 ###############################################################################
-FROM ghcr.io/keitaroinc/ckan:${CKAN_VERSION_RUNTIME_STAGE} as runtime
-
-ENV CKAN__PLUGINS "image_view text_view recline_view webpage_view datastore datapusher \
-  hierarchy_display hierarchy_form display_group relation \
-  spatial_metadata spatial_query datesearch repeating composite scheming_datasets \
-  password_policy clamav \
-  envvars"
+FROM ckan/ckan-base:${CKAN_VERSION_BUILD_STAGE} as runtime
 
-# Extra env for compatibility with ckan/base Docker images for downstream k8s
+ENV CKAN_DIR=${SRC_DIR}/ckan
+ENV DATA_DIR=/srv/app/data
 ENV CKAN_INI=${APP_DIR}/production.ini
-ENV CKAN_STORAGE_PATH=/var/lib/ckan
-ENV TZ="UTC"
+ENV UWSGI_HARAKIRI=50
 
 USER root
 
-# Install any system packages necessary to build extensions
-RUN set -ex && \
-  apt-get update && \
-  apt-get install -y --no-install-recommends \
-    clamav \
-    clamav-daemon \
-    libxml2-dev libxslt1-dev libgeos-c1v5 && \
-    pip install --no-cache-dir -U pip && \
-    rm -rf /var/lib/apt/lists/*
-
-# Copy python wheels from build stage
+# Setting the locale
+ENV LC_ALL="en_US.UTF-8"
+RUN update-locale LANG=${LC_ALL}
+
+# Update the package lists and install required packages
+RUN apt-get update && apt-get install -y \
+    bash \
+    gettext \
+    curl \
+    unzip \
+    libmagic1 \
+    libpcre3 \
+    libxslt1.1 \
+    libxml2 \
+    tzdata \
+    apache2-utils \
+    musl-dev \
+    libssl-dev \
+    proj-bin \
+    libproj-dev \
+    proj-data \
+    python3-cffi \
+    uwsgi-plugin-python3
```

Comment:
Do we need the `*-dev` version of those packages? If possible, use the non dev version.

Reply:
Correct me if I am wrong, but `-dev` version of the packages is used because some CKAN extensions need to build/compile against the packages such as musl, libssl, libproj etc, not because those packages cannot be used for production.

Those packages are used in the official Dockerfile for the lower CKAN versions as well https://github.com/ckan/ckan-docker-base/blob/main/ckan-2.10/Dockerfile#L49-L79.